app/command.go

```diff
@@ -202,7 +202,7 @@ func (a *App) ExecuteCommand(c request.CTX, args *model.CommandArgs) (*model.Com
 
 	args.TriggerId = triggerId
 
-	// Plugins can override built in and custom commands
+	// Plugins can override built in, custom, and product commands
```

Comment:
proposed order is:

plugins commands > products commands > custom commands > builtin commands

meaning that the items on the left can override items on the right.